tests/generic/baker_recipes.py

```diff
@@ -72,6 +73,8 @@
     breed="Super basset",
 )
 
+paulo_freire_school = Recipe(School, name="Escola Municipal Paulo Freire")
```

Comment:
🤓 ❤️

Reply:
model bakery has the best code easter eggs